RadixWallet/Features/AccountPreferencesFeature/Children/UpdateAccountLabel+View.swift

```diff
@@ -31,51 +32,78 @@ extension UpdateAccountLabel {
 		let sanitizedName: NonEmptyString?
 		let updateButtonControlState: ControlState
 		let hint: Hint?
+		let textFieldFocused: Bool
 	}
 
 	@MainActor
 	public struct View: SwiftUI.View {
 		let store: StoreOf<UpdateAccountLabel>
+		@Environment(\.dismiss) var dismiss
+		@FocusState private var textFieldFocus: Bool
 
 		init(store: StoreOf<UpdateAccountLabel>) {
 			self.store = store
 		}
 
 		public var body: some SwiftUI.View {
+			content
+				.withNavigationBar {
+					dismiss()
+				}
+				.presentationDetents([.fraction(0.5)])
```

Comment:
Maybe use `0.55`? Suggesting that because I chose `0.55` for the "Add New Gateway" screen, and it looks very similar to this one.

| 0.5 | 0.55 |
| - | - |
|![IMG_0041](https://github.com/user-attachments/assets/67b5dc0b-1016-49cb-8bb9-357e313dbb20) | ![IMG_0042](https://github.com/user-attachments/assets/b2d99179-6148-4247-af0b-99b9f50cc10a)|